### PR TITLE
Add installation section

### PR DESCRIPTION
**Pull request by aweiteka**
Also minor edits to .gitignore and requirements.txt

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,14 +50,48 @@ The `tests.conf` file contains configuration telling the CTF what extra remote t
...
 * git
 * behave >= 1.2.5
 * ansible
 
+### Installation
+
+1. Clone repositories
+```
+git clone https://github.com/Containers-Testing-Framework/ctf-cli.git
+git clone https://github.com/Containers-Testing-Framework/common-steps.git
```
Comment (vrutkovs) [inline README.md:65]: Cloning common-steps and common-features is not required - it would be performed during cli-launch

**Comment by vrutkovs** [comment]
Looks good to me, except the unnecessary cloning part

**Comment by thozza** [comment]
I think we can merge this as is and just add note that cloning common-steps/features is not necessary if one does not want to develop new common-steps/features.

@vrutkovs do you agree?

**Comment by vrutkovs** [comment]
Yeah, makes sense, we can postpone docs update until some milestone/release

**Pull request merged**